README.md

```diff
@@ -1,2 +1,10 @@
 # Artwork for the extendr project
 
+## Licensing terms for logos
+
+The extendr logo and hexlogo are distributed under the terms of the [Creative Commons Attribution-ShareAlike 4.0 International license](https://creativecommons.org/licenses/by-sa/4.0/) (CC-BY-SA 4.0).
+
+## Attribution
+
+* [The R logo](https://www.r-project.org/logo/) by The R Foundation is licensed under [CC-BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/).
+* [The Rust logo](https://github.com/rust-lang/rust-artwork) by Mozilla is licensed munder [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/).
```

Comment:
Quick note: at some point we'll probably need to replace "Mozilla" with "Rust Foundation"

https://github.com/rust-lang/foundation-faq-2020/blob/main/FAQ.md#q-trademark

Reply:
Could you open an issue for this?

Reply:
Here it is: #2